### PR TITLE
hardening/ErrorPutAttributes

### PR DESCRIPTION
**Pull request by rbl219**
Resolved issue  #1320

### CODE REVIEW

**Comment by fgalan** [comment]
LGTM

File: src/lib/serviceRoutinesV2/putEntity.cpp [hunk 1]
```diff
@@ -60,6 +61,7 @@ std::string putEntity
   ParseData*                 parseDataP
 )
 {
+  std::string answer = "";
   Entity*  eP = &parseDataP->ent.res;
```
Comment (kzangeli) [inline src/lib/serviceRoutinesV2/putEntity.cpp]: style guide: align eP to answer, and also the '='s

Comment (rbl219) [inline src/lib/serviceRoutinesV2/putEntity.cpp]: Fix on commit 3d97cf1

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**